### DISCUSSION TITLE
What is the risk of permanent account lockout due to SMS 2FA?

### DISCUSSION THREAD

**Discussion by SoftCircuits** [community/community#133973] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Why am I getting these messages saying that I need to configure another 2FA method to reduce my rick of permanent account lockout?

Regardless of how secure SMS 2FA might be (or not), why would I be permanently locked out of my account?

**Comment by 0x-Apollyon** [comment]
Incase you lose your phone or your device gets compromised.
SMS 2fa isnt really useful or secure in 2024 anymore

**Comment by davevad93** [comment]
Hi @SoftCircuits ,

A couple of weeks ago, someone from the GitHub staff has given an official answer about this 2FA thing. Here's the link: https://github.com/orgs/community/discussions/129244#discussioncomment-10024978

As you can see in the link I've provided, the reason why you can be permanently locked out from your account if you use SMS 2FA hasn't been explained properly either. I guess since it's an insecure method because SMS aren't encrypted and are vulnerable to several types of attacks, in the event that someone hacks your account you would lose access to it (?) , so basically they are encouraging users to use more secure methods like the GitHub mobile app or an authenticator app.

These are just suppositions to be fair, so I don't know what the real reason is.
